Initial ru lang for Susy + Quest

marisathewitch opened the pull request:
Currently translated at 6.1% (134 of 2181 strings) for core mod
Initial lang files for QuestBook

marisathewitch committed:
Create ru_ru.lang

Commit:
Translated using Weblate (Russian)

Currently translated at 1.9% (43 of 2175 strings)

Translation: Supersymmetry/Supersymmetry
Translate-URL: http://weblate.pantsu.moe/projects/supersymmetry/supersymmetry/ru/

marisathewitch committed:
Merge branch 'SymmetricDevs:master-ceu' into master-ceu

Commit:
Added translation using Weblate (Russian)

Commit:
Translated using Weblate (Russian)

Currently translated at 0.1% (1 of 1822 strings)

Translation: Supersymmetry/Questbook
Translate-URL: http://weblate.pantsu.moe/projects/supersymmetry/questbook/ru/

Commit:
Added translation using Weblate (Russian)

marisathewitch committed:
delete ghost folder

marisathewitch committed:
Merge pull request #1 from marisathewitch/test-ru

delete ghost folder

Commit:
Translated using Weblate (Russian)

Currently translated at 0.1% (1 of 1822 strings)

Translation: Supersymmetry/Questbook
Translate-URL: http://weblate.pantsu.moe/projects/supersymmetry/questbook/ru/

Commit:
Translated using Weblate (Russian)

Currently translated at 4.3% (94 of 2175 strings)

Translation: Supersymmetry/Supersymmetry
Translate-URL: http://weblate.pantsu.moe/projects/supersymmetry/supersymmetry/ru/

Commit:
Translated using Weblate (Russian)

Currently translated at 0.1% (3 of 1822 strings)

Translation: Supersymmetry/Questbook
Translate-URL: http://weblate.pantsu.moe/projects/supersymmetry/questbook/ru/

marisathewitch committed:
Merge branch 'SymmetricDevs:master-ceu' into master-ceu

Commit:
Translated using Weblate (Russian)

Currently translated at 5.0% (111 of 2181 strings)

Translation: Supersymmetry/Supersymmetry
Translate-URL: http://weblate.pantsu.moe/projects/supersymmetry/supersymmetry/ru/

Commit:
Translated using Weblate (Russian)

Currently translated at 6.1% (134 of 2181 strings)

Translation: Supersymmetry/Supersymmetry
Translate-URL: http://weblate.pantsu.moe/projects/supersymmetry/supersymmetry/ru/

marisathewitch committed:
Merge branch 'SymmetricDevs:master-ceu' into master-ceu

loxoDev commented:
Do you want this to be merged yet, or is this just so other people can see the progress?

Commit:
Translated using Weblate (Russian)

Currently translated at 6.1% (135 of 2182 strings)

Translation: Supersymmetry/Supersymmetry
Translate-URL: http://weblate.pantsu.moe/projects/supersymmetry/supersymmetry/ru/

marisathewitch commented:
> Do you want this to be merged yet, or is this just so other people can see the progress?

Leave it as an indicator of progress for now.
There's nothing to add to the game yet. There is too little progress.

marisathewitch committed:
Merge branch 'SymmetricDevs:master-ceu' into master-ceu

loxoDev commented:
Okay, thank you. Just ping me on discord once you feel like its ready.

Commit:
Translated using Weblate (Russian)

Currently translated at 6.5% (142 of 2182 strings)

Translation: Supersymmetry/Supersymmetry
Translate-URL: http://weblate.pantsu.moe/projects/supersymmetry/supersymmetry/ru/

Commit:
Translated using Weblate (Russian)

Currently translated at 7.1% (155 of 2182 strings)

Translation: Supersymmetry/Supersymmetry
Translate-URL: http://weblate.pantsu.moe/projects/supersymmetry/supersymmetry/ru/

Commit:
Translated using Weblate (Russian)

Currently translated at 7.2% (159 of 2182 strings)

Translation: Supersymmetry/Supersymmetry
Translate-URL: http://weblate.pantsu.moe/projects/supersymmetry/supersymmetry/ru/

Commit:
Translated using Weblate (Russian)

Currently translated at 7.9% (174 of 2182 strings)

Translation: Supersymmetry/Supersymmetry
Translate-URL: http://weblate.pantsu.moe/projects/supersymmetry/supersymmetry/ru/

Commit:
Translated using Weblate (Russian)

Currently translated at 11.8% (259 of 2182 strings)

Translation: Supersymmetry/Supersymmetry
Translate-URL: http://weblate.pantsu.moe/projects/supersymmetry/supersymmetry/ru/

Commit:
Translated using Weblate (Russian)

Currently translated at 12.2% (267 of 2182 strings)

Translation: Supersymmetry/Supersymmetry
Translate-URL: http://weblate.pantsu.moe/projects/supersymmetry/supersymmetry/ru/

marisathewitch committed:
Merge branch 'SymmetricDevs:master-ceu' into master-ceu

Commit:
Translated using Weblate (Russian)

Currently translated at 12.3% (269 of 2182 strings)

Translation: Supersymmetry/Supersymmetry
Translate-URL: http://weblate.pantsu.moe/projects/supersymmetry/supersymmetry/ru/

Commit:
Translated using Weblate (Russian)

Currently translated at 12.8% (280 of 2182 strings)

Translation: Supersymmetry/Supersymmetry
Translate-URL: http://weblate.pantsu.moe/projects/supersymmetry/supersymmetry/ru/

marisathewitch committed:
Merge branch 'SymmetricDevs:master-ceu' into master-ceu

Commit:
Translated using Weblate (Russian)

Currently translated at 13.0% (285 of 2182 strings)

Translation: Supersymmetry/Supersymmetry
Translate-URL: http://weblate.pantsu.moe/projects/supersymmetry/supersymmetry/ru/

Commit:
Translated using Weblate (Russian)

Currently translated at 1.4% (26 of 1822 strings)

Translation: Supersymmetry/Questbook
Translate-URL: http://weblate.pantsu.moe/projects/supersymmetry/questbook/ru/

marisathewitch committed:
Merge branch 'SymmetricDevs:master-ceu' into master-ceu

Commit:
Translated using Weblate (Russian)

Currently translated at 13.4% (303 of 2258 strings)

Translation: Supersymmetry/Supersymmetry
Translate-URL: http://weblate.pantsu.moe/projects/supersymmetry/supersymmetry/ru/

Commit:
Translated using Weblate (Russian)

Currently translated at 14.4% (327 of 2258 strings)

Translation: Supersymmetry/Supersymmetry
Translate-URL: http://weblate.pantsu.moe/projects/supersymmetry/supersymmetry/ru/

Commit:
Translated using Weblate (Russian)

Currently translated at 16.3% (370 of 2258 strings)

Translation: Supersymmetry/Supersymmetry
Translate-URL: http://weblate.pantsu.moe/projects/supersymmetry/supersymmetry/ru/

Commit:
Translated using Weblate (Russian)

Currently translated at 16.6% (376 of 2258 strings)

Translation: Supersymmetry/Supersymmetry
Translate-URL: http://weblate.pantsu.moe/projects/supersymmetry/supersymmetry/ru/

marisathewitch committed:
Merge branch 'SymmetricDevs:master-ceu' into master-ceu

marisathewitch committed:
Merge branch 'SymmetricDevs:master-ceu' into master-ceu

JennyDoll commented on config/betterquesting/resources/supersymmetry/lang/ru_ru.lang:
```diff
@@ -0,0 +1,28 @@
+
+
+susy.quest.db.0.desc=Добро пожаловать в §9§9§9§b§b§b§b§b§b§l§l§l§l§l§lSupersymmetry§r§r§r§r§r§r!%n%nВы приземлились на неизвестную планету. Эта книга научит вас, как восстановить цивилизацию с нуля всего за §k4200§r страниц.%n%nНачните со сбора камней§r. Эта книга заданий интерактивная.%n%nКогда вы соберёте камни с земли, нажмите кнопку §l§l§l§l§l§l§a§a§a§a§a§aОбнаружить/Отправить§r чтобы заверши ...
```
> не доступны

недоступны*

Может быть, стоит заменить "GRAVE (`)" на "Ё"? Обыкновенно на этом месте на клавиатурах с кириллицей Ё/тильда.

JennyDoll commented on config/betterquesting/resources/supersymmetry/lang/ru_ru.lang:
```diff
@@ -0,0 +1,28 @@
+susy.quest.db.0.title=Основы
+susy.quest.db.1.title=Это JEI, не NEI. Используй с умом.
+susy.quest.db.88.desc=Убив Корову вы получите §6Кожу§r. Она может быть использована для изготовления Простейшей брони.%n%nОтметим что животные не очень часто встречаются, но вы всё ещё можете попытаться отыскать их.
+susy.quest.db.2.desc=Вы можете выращивать некоторые культуры только в определенное время года (но вы можете изменить это с помощью §aТеплицы§r или автоматизированного земледелия), и окружающая среда вокруг вас будет меняться в течение года.%n%n§aВесна §rначинается 1 Марта и заканчивается 30 Мая.%n§6Лето §rначинается 1 Июля и заканчивается 30 Августа.%n§cОсень §rначинается 1 Сентября и заканчивает ...
```
Названия месяцев в русском языке пишутся с маленькой буквы, насколько мне известно. 
https://gramota.ru/spravka/vopros/287081

JennyDoll commented on config/betterquesting/resources/supersymmetry/lang/ru_ru.lang:
```diff
@@ -0,0 +1,28 @@
+susy.quest.db.4.desc=Том Акаши это сборник нескольких других руководств, которые могут быть вам полезны. Вы можете получить его бесплатно, поставив галочку.
+susy.quest.db.5.desc=Ваш новый §eКремниевый нож §r можно использовать для резки вещей,§f превращения их в осколки или получения §aРастительных волокон§..%n%nЧтобы разрезать предметы, кроме блоков, вам нужно держать предмет, который вы хотите разрезать, в §0§aлевой руке§r и нож в §aосновной(правой) руке.§r Это работает и наоборот.%n%nВы также можете использовать Кремниевый нож в качестве оружия.
+susy.quest.db.8.desc=Вы можете использовать кремень для разработки нового §6оружия§r. Все, что вы можете сделать прямо сейчас, это лук и стрелы, но этого пока вам хватит.%n%n
+susy.quest.db.3.desc=Осколки кремня§r необходимы для получения инструментов.%n%nГравий можно добывать без инструмента, и вы можете получить кремень.%n%nКремень можно расколоть [§aЩелкните правой кнопкой мыши, глядя на верхнюю сторону камня§r] на открытой поверхности камня, чтобы получить осколки кремня.
```
> Гравий можно добывать без инструмента, и вы можете получить кремень.

Косноязычно звучит. В оригинале, если честно, тоже. Что-то в духе "Кремень можно получить из гравия, для добычи которого не нужны инструменты", наверное? Тоже кривовато звучит, но лучше, чем околомашинный перевод...

JennyDoll commented on config/betterquesting/resources/supersymmetry/lang/ru_ru.lang:
```diff
@@ -0,0 +1,28 @@
+susy.quest.db.5.desc=Ваш новый §eКремниевый нож §r можно использовать для резки вещей,§f превращения их в осколки или получения §aРастительных волокон§..%n%nЧтобы разрезать предметы, кроме блоков, вам нужно держать предмет, который вы хотите разрезать, в §0§aлевой руке§r и нож в §aосновной(правой) руке.§r Это работает и наоборот.%n%nВы также можете использовать Кремниевый нож в качестве оружия.
+susy.quest.db.8.desc=Вы можете использовать кремень для разработки нового §6оружия§r. Все, что вы можете сделать прямо сейчас, это лук и стрелы, но этого пока вам хватит.%n%n
+susy.quest.db.3.desc=Осколки кремня§r необходимы для получения инструментов.%n%nГравий можно добывать без инструмента, и вы можете получить кремень.%n%nКремень можно расколоть [§aЩелкните правой кнопкой мыши, глядя на верхнюю сторону камня§r] на открытой поверхности камня, чтобы получить осколки кремня.
+susy.quest.db.9.desc=Используя ваш §6Верстак§r вы можете создать более лучшие инструменты, §6Кремневую кирку§r для добычи руды и §6Кремневую лопату§r для копания песка или гравия.%n%nПосле получения большого количества кремня вы можете заняться созданием более лучшего §6Кремневого топора§r, который срубит целое дерево одним ударом, и лучшей §6Кремневой кирки§r, которая автоматически размещает фак ...
```
> более лучшие

Нет, это неправильно образованная превосходная степень. "более качественные" или, если всё совсем грустно, "более хорошие". 

UPD: Или "более продвинутые".

JennyDoll commented on config/betterquesting/resources/supersymmetry/lang/ru_ru.lang:
```diff
@@ -0,0 +1,28 @@
+susy.quest.db.8.desc=Вы можете использовать кремень для разработки нового §6оружия§r. Все, что вы можете сделать прямо сейчас, это лук и стрелы, но этого пока вам хватит.%n%n
+susy.quest.db.3.desc=Осколки кремня§r необходимы для получения инструментов.%n%nГравий можно добывать без инструмента, и вы можете получить кремень.%n%nКремень можно расколоть [§aЩелкните правой кнопкой мыши, глядя на верхнюю сторону камня§r] на открытой поверхности камня, чтобы получить осколки кремня.
+susy.quest.db.9.desc=Используя ваш §6Верстак§r вы можете создать более лучшие инструменты, §6Кремневую кирку§r для добычи руды и §6Кремневую лопату§r для копания песка или гравия.%n%nПосле получения большого количества кремня вы можете заняться созданием более лучшего §6Кремневого топора§r, который срубит целое дерево одним ударом, и лучшей §6Кремневой кирки§r, которая автоматически размещает фак ...
+susy.quest.db.10.desc=Под землей есть много полезных ресурсов, поэтому вам следует поискать их.%n%nВам нужно искать §l§aМагнетит§r. Позже он будет использован при создании магнитов.%n%nМагнетит встречается в рудных месторождениях§а §rпод землей. Месторождения руды имеют размер около 20 х 20 блоков.%n%nЕсли вы увидите крошечная кучку магнетита на поверхности, это будет означать, что под землей ест ...
```
> вам следует поискать их

Лучше "вам следует заняться их поиском", но это вкусовщина.

> крошечная кучку магнетита

Крошечную.

JennyDoll commented on config/betterquesting/resources/supersymmetry/lang/ru_ru.lang:
```diff
@@ -0,0 +1,28 @@
+susy.quest.db.3.desc=Осколки кремня§r необходимы для получения инструментов.%n%nГравий можно добывать без инструмента, и вы можете получить кремень.%n%nКремень можно расколоть [§aЩелкните правой кнопкой мыши, глядя на верхнюю сторону камня§r] на открытой поверхности камня, чтобы получить осколки кремня.
+susy.quest.db.9.desc=Используя ваш §6Верстак§r вы можете создать более лучшие инструменты, §6Кремневую кирку§r для добычи руды и §6Кремневую лопату§r для копания песка или гравия.%n%nПосле получения большого количества кремня вы можете заняться созданием более лучшего §6Кремневого топора§r, который срубит целое дерево одним ударом, и лучшей §6Кремневой кирки§r, которая автоматически размещает фак ...
+susy.quest.db.10.desc=Под землей есть много полезных ресурсов, поэтому вам следует поискать их.%n%nВам нужно искать §l§aМагнетит§r. Позже он будет использован при создании магнитов.%n%nМагнетит встречается в рудных месторождениях§а §rпод землей. Месторождения руды имеют размер около 20 х 20 блоков.%n%nЕсли вы увидите крошечная кучку магнетита на поверхности, это будет означать, что под землей ест ...
+susy.quest.db.1.desc=Вероятно, невозможно играть в GregTech без использования JEI. Итак, вот несколько советов о том, как использовать JEI в полной мере.%n%nЭтот модпак предполагает, что IQ игрока выше 100. Если вы запутались, нажмите §6R §rна предмете в книге заданий, чтобы увидеть рецепт создания предмета, или §6U§r, чтобы узнать, как он используется.%n%n§eРегулярно добавляйте предметы в заклад ...
```
> Крохотной кучки пыли

Лучше унифицировать перевод, чтобы не было в разных местах отдельно "крохотной" и "крошечной".

> Нефтепереработка и ядерная химия довольно плохо с этим справляются.

С чем?
"Нефтепереработка и ядерная химия больше остальных отраслей требуют внимания к рецептам", или, как мне подсказывают, "требуют повышенного внимания". В принципе стоит перестроить структуру последних двух предложений; несмотря на то, что они переведены достоверно, они не адаптируют речь к разговорному русскому языку. Здесь, скорее, подойдёт сложноподчинённое предложение.

JennyDoll reviewed (commented):
Перевод очень машинный.

marisathewitch commented:
> Перевод очень машинный. 

Буду исправлять. Спасибо за заметки

Commit:
Translated using Weblate (Russian)

Currently translated at 22.3% (503 of 2247 strings)

Translation: Supersymmetry/Supersymmetry
Translate-URL: http://weblate.pantsu.moe/projects/supersymmetry/supersymmetry/ru/

Commit:
Translated using Weblate (Russian)

Currently translated at 26.5% (597 of 2247 strings)

Translation: Supersymmetry/Supersymmetry
Translate-URL: http://weblate.pantsu.moe/projects/supersymmetry/supersymmetry/ru/

Commit:
Translated using Weblate (Russian)

Currently translated at 28.9% (651 of 2247 strings)

Translation: Supersymmetry/Supersymmetry
Translate-URL: http://weblate.pantsu.moe/projects/supersymmetry/supersymmetry/ru/

marisathewitch committed:
Merge branch 'SymmetricDevs:master-ceu' into master-ceu
